eng/Publishing.props

```diff
@@ -98,11 +98,16 @@
       <IsStableBuild>false</IsStableBuild>
       <IsStableBuild Condition="'$(DotNetFinalVersionKind)' == 'release'">true</IsStableBuild>
     </PropertyGroup>
-    
+
+    <!-- In VMR inner-build copy to local storage, do not push to AzDO -->
+    <PropertyGroup Condition="'$(ArcadeInnerBuildFromSource)' == 'true' or '$(DotNetBuildPhase)' == 'InnerRepo'">
```

Comment:
You can remove ArcadeInnerBuildFromSource, the legacy control set isn't in use in this repo.

Reply:
Will do - thanks.

Reply:
Fixed with https://github.com/dotnet/sdk/pull/39311/commits/7a090920861b59ad6875841ecdb1d6f0081d29c6